frontend/src/components/common/Badge/Badge.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import { BadgeSize } from '../../../utils';
+
+export interface IBadgeProps {
+  value: number;
+  size: BadgeSize;
+}
+
+const Badge: FC<IBadgeProps> = ({ ...props }) => {
+  const { value, size } = props;
+  return (
+    <>
+      {value ? (
+        <span
+          className={`
+          ${size === 'small' ? 'h-6 w-6 text-sm ' : ''}
+          ${size === 'middle' ? 'h-7 w-7 text-base ' : ''}
+          ${size === 'large' ? 'h-8 w-8 text-lg ' : ''}
+          mx-2 flex items-center justify-center rounded-lg text-white primary-color-bg`}
+        >
+          {value}
+        </span>
+      ) : (
+        ''
+      )}
```

Comment:
I'd use the shorthand (value && <...>) instead of the ternary operator and I'd suggest making another map like the one in the box for assigning the classes dynamically...

Reply:
Fixed [here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/05024ac6bb3c83f5f16a1ae6a11884fe8ed50105)